bats/tests/profile/deployment.bats

```diff
@@ -250,11 +250,18 @@ api_set() {
 @test 'try to change locked fields via rdctl start' {
     rdctl start --container-engine.allowed-images.enabled=false --no-modal-dialogs
     try --max 10 --delay 5 assert_file_contains "$PATH_LOGS/background.log" 'field "containerEngine.allowedImages.enabled" is locked'
-    assert_success
+
+    # Make sure that Rancher Desktop has really stopped; otherwise `rdctl start` may not launch a new instance
+    if is_macos; then
```

Comment:
Looks fine. You might want to add a comment that the two `rdctl start` commands should fail to stand up rancher-desktop, because of the locked-field violation.

Reply:
I've added it to the test description, and also a comment. But this really isn't related to the PR, but should have been part of the test already.